packages/npm-registry-agent/test/index.ts

```diff
@@ -22,39 +21,37 @@ const OPTS = {
   key: 'key',
   localAddress: 'localAddress',
   maxSockets: 5,
-  strictSSL: 'strictSSL',
+  strictSSL: true,
```

Comment:
After switching `agent` to be an `import`, typescript was able to recognize that the value of `strictSSL` should have been a boolean. Updated test assertions of `rejectUnauthorized` to as well.

`rejectUnauthorized` is also a boolean according to Node docs.
https://nodejs.org/api/tls.html#tls_tls_connect_options_callback